doc/quickstart/run.rst

```diff
@@ -6,32 +6,57 @@ Running
 The ESMValCore package provides the ``esmvaltool`` command line tool, which can
 be used to run a :doc:`recipe <../recipe/index>`.
 
-To run a recipe, call ``esmvaltool run`` with the desired recipe:
+To list the available commands, run
 
 .. code:: bash
 
-	esmvaltool run recipe_python.yml
+    esmvaltool --help
+
+It is also possible to get help on specific commands, e.g.
+
+.. code:: bash
+
+    esmvaltool run --help
+
+will display the help message with all options for the ``run`` command.
+
+To run a recipe, call ``esmvaltool run`` with the path to the desired recipe:
+
+.. code:: bash
+
+	esmvaltool run recipe_example.yml
+
+The ``esmvaltool run recipe_example.yml`` command will first look if
```

Comment:
here too, we should probably use the name of an existing recipe or the vaguely geeky `$RECIPE_NAME` - some users will actually try running with `recipe_example.yml` :grin:

Reply:
I'll add a note that this is just an example and the recipe does not actually exist. I would prefer not to use a recipe from ESMValTool because that may not even be installed.

Reply:
good call, how about we copy `recipe_python.yml` in Core - that'll run without actually installing Tool

Reply:
It wouldn't work, because you'd also need the diagnostic script and all the shared code that the diagnostic script uses